Resources/Prototypes/Entities/Objects/Fun/toys.yml

```diff
@@ -504,7 +504,7 @@
   - type: Item
     size: 24
```

Comment:
On second thought, this needs removed, let it inherit the parent BaseItem size of 5 so remove the two lines

Reply:
I think it still needs the Item component so we can use HeldPrefix, removed size though and it inherits 5 properly